src/components/Form/Select/style.postcss

```diff
@@ -1,22 +1,38 @@
 @import 'main.postcss';
 
-.Select {
-  display: flex;
-  margin-bottom: 1em;
+:root {
+  --arrowAreaSize: calc(var(--touchableMinHeightNew) / 2);
+  --arrowSize: 9px;  /* 9px x 5px */
```

Comment:
Just a small thing, but that comment there seems to be inconsistent with the actual value.